CMakeLists.txt

```diff
@@ -590,6 +590,10 @@ if(BUILD_ONNX_PYTHON)
     target_link_libraries(onnx_cpp2py_export
                           PRIVATE "-Wl,--whole-archive" $<TARGET_FILE:onnx>
                                   "-Wl,--no-whole-archive")
+    # Prevent "undefined symbol: _ZNSt10filesystem7__cxx114path14_M_split_cmptsEv" on gcc 8
```

Comment:
```suggestion
    # Prevent "undefined symbol: _ZNSt10filesystem7__cxx114path14_M_split_cmptsEv" 
    # (std::filesystem::__cxx11::path::_M_split_cmpts()) on gcc 8
```

Reply:
On x86 hosts, `pip install onnx` will use our prebuilt wheels which don't show the issue. You'd need to do `pip3 install --no-binary onnx onnx` so it builds from the tar.gz. Also the problem won't appear until you attempt `import onnx` after it's installed.

The easiest way to replicate might be to use use a container. The following Dockerfile should exhibit the issue even on a x86 machine. (Note, you'd have to run this on a RHEL host because the Dockerfile uses `yum` to install some base packages. So building the image might require the host building it to have an active RedHat subscription)
```
FROM registry.access.redhat.com/ubi8/ubi:latest

RUN yum install -y gcc-c++ make cmake python39-devel
RUN pip3 install --no-binary onnx onnx

RUN python3 -c "import onnx"
```